### PR TITLE
Ft/blog section

### PR DESCRIPTION
**Pull request by mhenriette**
## Describe your changes
ID-> #025
Link -> [here](https://www.notion.so/apeunit/Blog-section-6a13ba5ca2a143a7ad651e59462750dd)

## Issue ticket number and link
created Blog component in the homepage section folder. 
created Blog cards with tag at top , title, paragraph and section of tags (use tag component from UI folder) with background.
## Tasks completed

- [x] Created Blog Section in Homepage folder
- [x] Created BlogCard file in ui folder
- [x] Added card content in the BlogCard using Data created in blogCard file in Data Folder
- [x] added spacing in tailwind config file
## Tasks not completed
none

### CODE REVIEW

File: components/ui/BlogCard.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            <h2>{title}</h2>
+            <p>{subtitle}</p>
+          </div>
+      </div>
+       
+      <div className='flex flex-wrap gap-x-6 gap-y-2 items-center'>
+        {tags.map(el=><Tag key={el.id} fill>{el.name}</Tag>)}
+       
+      
+
```
Comment (hirwaaldo1) [inline components/ui/BlogCard.jsx]: remove space

File: data/blogCard.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        } ,
+        { 
+            id:3,
+            name:'Decentralization'
+        
+        }   ,
+        { 
+            id:4,
+            name:'Blockchain'
+
```
Comment (hirwaaldo1) [inline data/blogCard.js]: remove the spaces

**Review by hirwaaldo1 — CHANGES_REQUESTED** [review]
remove space it will be great

**Review by hirwaaldo1 — APPROVED** [review]
well done

**Review by mutesialine — APPROVED** [review]
Well done

**Pull request merged**